src/endpoints/createOffersEndpoint.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import OfferModel from '../models/OfferModel'
+import EndpointBuilder from '../EndpointBuilder'
+
+import ParamMissingError from '../errors/ParamMissingError'
+import type { JsonOfferPostType, JsonOfferType } from '../types'
+import Endpoint from '../Endpoint'
+
+export const OFFERS_ENDPOINT_NAME = 'extras'
+
+const createPostMap = (jsonPost: JsonOfferPostType): Map<string, string> => {
```

Comment:
This can be simplified to: `new Map(Object.entries(jsonPost))`

Reply:
Flow doesn't like that yet. See [here](https://github.com/facebook/flow/issues/5838).

Reply:
Should we use here a @FlowFixMe?
Or is it correct that the type could be mixed?